### ISSUE TITLE
python 3.8 incompatibility on Conda?

### ISSUE THREAD

**Issue by Blosberg** [FelixKrueger/TrimGalore#99]
Thanks for this excellent resource.

I've just tried to run `$ conda install trim-galore`, (with bioconda included in my channels) and I encounter the following error:

```

Found conflicts! Looking for incompatible packages.
... UnsatisfiableError: The following specifications were found
to be incompatible with the existing python installation in your environment:

Specifications:

  - trim-galore -> python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.5,<3.6.0a0|3.4.*']

  Your python: python=3.8

  If python is on the left-most side of the chain, that's the version you've asked for.
  When python appears to the right, that indicates that the thing on the left is somehow
  not available for the python version you are constrained to. Note that conda will not
  change your python version to a different minor version unless you explicitly specify
  that.
```
From checking [this issue here](https://github.com/FelixKrueger/TrimGalore/issues/78) it seems like python 3 should be ok, and it seemed odd to me that trim_galore's compatibility would go up to but not including 3.8. Should I check out an earlier version of python3?

**Comment by FelixKrueger** [comment]
Hi Brendan,

Python is required for running Cutadapt, but I can't imagine why Python 3.8 would not work (with Cutadapt). Maybe the Conda recipe is a bit overly strict?

**Comment by Blosberg** [comment]
Hi Felix. Thanks for the suggestion.
I got the same error with `$conda install cutadapt` , so indeed that's where the problem was. I don't have enough experience with Conda recipe's to fix that manually, so instead, working backwards, I started a clean environment with _just_ trim_galore, and then installed all my other dependencies one by one, waiting to see when the same conflict would emerge .... except, strangely, that never happened. All my packages are now installed. 
I can't say I really understand how that's possible. My best guess is that it's because I started the environment with `$conda create -n [name] python`, which (I'm guessing) fixed the most recent python version (3.8).  In any case, it's working now. Thanks for your help.
